AsyncDisplayKit/ASTableView.h

```diff
@@ -94,6 +94,7 @@
  */
 - (void)beginUpdates;
 - (void)endUpdates;
+- (void)endUpdatesAnimated:(BOOL)animated completion:(void (^)(BOOL completed))completion;
```

Comment:
I like how you've structured this.  A single more detailed option that provides the completion, even if you're not interested in animation (just pass NO).  Worth an AppleDoc comment with a brief explanation of the semantics of the completion?